### ISSUE TITLE
Add Non-Microsoft Representative(s) To Review Board Sooner Rather Than Later

### ISSUE THREAD

**Issue by mletterle** [Microsoft/xaml-standard#118]
The [Review Board](https://github.com/Microsoft/xaml-standard/blob/staging/docs/reviewboard.md) Document currently states:

> For XAML Standard 1.0 - the primary goal is to unify the dialects between UWP and Xamarin.Forms. So, for the most part, the decision maker is Microsoft with input on what is desired from the broader community here.
>
> Going forward, we want to expand the review board as we grow: as more platform vendors and API drivers appear, and as we get more implementers of the standard, we will grow this review board as well.

If there is truly to be a `XAML Standard` it should include non-Microsoft/Xamarin representatives from the start, even if it just happens to be a "community representative" rather than a vendor of some sort (though I believe there are appropriate vendors already out there, [noesisengine](http://www.noesisengine.com/) comes to the top of my mind, as does [Ammy](http://www.ammyui.com/)

These third parties should be involved from the start rather than attempted to be bolted on later. 

Pull Request #111 might be related to some extent.

**Comment by Mike-EEE** [comment]
Excellent.

**Comment by insinfo** [comment]
@mletterle
I fully agree with you, if it is an open standard, it has to happen in this way that you quoted, and I would say more, Microsoft and the team involved, should make a formal invitation to companies that hold much of the market as: Adobe who developed the Adobe XD is a software that should embrace XAML Standard, among others such as Bohemian who developed Sketchapp, among many other companies that develope designer tools and prototyping tools

**Comment by dotMorten** [comment]
It might be worth looking at what .NET Standard does: https://github.com/dotnet/standard/tree/master/docs/review-board

Basically only implementers of the standard is part of the review board. I shouldn't confuse "Open Standard" with "Democracy". A standard does need some key stakeholders to keep things consistent, maintainable and extensible. Since the only parties currently needing to implement this is UWP and XF, it makes sense they are the review board - at least until other vendors - like noesis - comes along and want to join the party. Too many players aren't always a good thing. [OAuth 2.0 is a good example of what we don't want to happen](https://www.cnet.com/news/oauth-2-0-leader-resigns-says-standard-is-bad/).

An alternative might be a Advisory Board that works with the Review Board and push community and key industry agendas.

**Comment by monkeynoises** [comment]
I'd argue that there should be at least someone from .NET Core/.NET Standard team on the board. In the past the .NET team was in full control of the Full Framework, and all the related stacks...and so there was a holistic approach. You then had the reorg where some of the XAML team moved to UWP, some didn't. It's just with the new ideas in .NET Core/.NET Standard...some of those could be injected into the XAML Standard - it's not about them regaining control again.....they've got enough on their plate.

**Comment by birbilis** [comment]
this should get a "meta" tag
